### DISCUSSION TITLE
Presa smart da esterno

### DISCUSSION THREAD

**Discussion by SkunkGuru** [indomus/forum#2721] [GENERALE - Discussioni generiche]
Ciao a tutti,

mi consigliate una presa smart da esterno, per poter temporizzare da HomeAssistant l'accensione delle luci esterne ? internamente ho le ShellyPlugS e mi trovo bene; da esterno (quindi sotto pioggia e freddo) cosa si può usare che sia ovviamente integrabile in HA ?

purtroppo non posso montare uno shelly o similare nella cassettina, perchè con i lavori esterni hanno murato/cementato la presa e non si riesce a smontare senza far danno

Grazie 😄

**Comment by SkunkGuru** [comment]


**Reply by M0ntyC0r4** [reply]
di solito l'interruttore delle luci dell'esterno sta comunque all'interno (o comunque in balcone/veranda).
agisci sull'interruttore invece che sulle luci e montaci un sonoff, uno shelly..roba così...altrimenti devi comprare una scatola di derivazione esterna fatta apposta per i giardini/balconi, la monti e la presa smart la colleghi a quella.
